includes/sdk-troubleshooting-and-debugging/latest-android.md

```diff
@@ -0,0 +1,11 @@
+--8<-- "includes/sdk-troubleshooting-and-debugging/latest-sdk-how-to-debug.md"
+
+#### Plugins
+You can take advantage of the Destination Plugin to print out the configuration value and event payload before sending them to the server. Please follow the github plugin examples to understand how it works. 
```

Comment:
can we make this paragraph as an include file and share it with the one in latest ios?

Maybe something like
"
You can take advantage of a Destination Plugin to print out the configuration value and event payload before sending them to the server. You can set the log level to debug, copy a following Github plugin example into your project, and add the plugin into your Amplitude instance.
"

Reply:
old ios have a different way to enable the log and there is no log level in ios...